src/locales/en/messages.json

```diff
@@ -1565,6 +1565,9 @@
   "premiumSignUpTwoStep": {
     "message": "Additional two-step login options such as YubiKey, FIDO U2F, and Duo."
   },
+  "premiumEmergencyAccess": {
```

Comment:
Sorry, just reviewing the rest of these changes now, looks like this key doesn't match what you're referencing in the html component, `premiumSignUpEmergency` vs. `premiumEmergencyAccess`; I can't find the key `premiumSignUpEmergency` anywhere in the current messages.json file.

Reply:
I have updated it. Sorry for making this complicated.